Docs/ACPI/SSDT-Swap-Command and Option.dsl

```diff
@@ -0,0 +1,14 @@
+// This sample shows how to Swap command and option.
+DefinitionBlock ("", "SSDT", 2, "ACDT", "ps2", 0)
+{
+    External (_SB_.PCI0.LPCB.PS2K, DeviceObj)
+    
+    Name(_SB.PCI0.LPCB.PS2K.RMCF, Package()
+    {
+        "Keyboard", Package()
+        {
+            "Swap command and option", ">n", 
```

Comment:
Does this disable command and option swapping? This looks a bit counter-intuitive to me. Why is that enabled  _by default_? CC @usr-sse2 @zhen-zen

Reply:
the Italian keyboard for the @ you press alt gr + @ on windows, instead on macos corresponds to cmd + @, if I use this ssdt everything is back normal

Reply:
we use keyboards born for windows, not Apple keyboards, so you have to use it for Italian layout keyboard, I don't know for other layouts if you need to

Reply:
Yes, it's enabled by default and for English layout this function works ootb without any issues:  https://github.com/acidanthera/VoodooPS2/blob/096f654c986d8d135c0e422eb89e582136cf5736/VoodooPS2Keyboard/VoodooPS2Keyboard-Info.plist#L281

Reply:
in my opinion, it is useful for the rest of the layouts I do not propose modification to the plist but an ssdt for those who need it

Reply:
@zhen-zen I agree that it is a great feature, and I also expect it to work properly, but I believe it is counter-intuitive to have it enabled by default.

Reply:
Maybe that's the difference between a generic HID driver and one trying to emulate vanilla Apple keyboards' behavior. Current VoodooPS2 configuration and default mapping tries to imitate apple ones, so this option and "Make Application key into right windows" is enabled by default as well.

Reply:
I use this ssdt in my notebooks, I don't want to always change the setting to system preferences

Reply:
@zhen-zen I prefer Apple mapping myself, but that does not seem right to enforce it by default. I suugest disable it by default and include and SSDT to enable it in the samples. Do you agree? Also CC @usr-sse2

Reply:
I always wanted to disable it by default, just haven't done it. 
I prefer using keys for what they are designed and what's written on them, no matter where they are located on the keyboard.

Reply:
> I always wanted to disable it by default, just haven't done it.
> I prefer using keys for what they are designed and what's written on them, no matter where they are located on the keyboard.

I think of the many users who use this fix or do it from system preferences, so my thoughts go to them.

Reply:
I wouldn't touch the kext plist, just adding ssdt explaining what it's for.

Reply:
Personally I'm neutral on that. Users take it for granted that `PrtSc` is just disable touchpad and `Alt` for `Command` rather than `Option`. That prejudice have been around for a long time but they will eventually realize it after switching to other input devices.

@Lorys89 Just change the plist value quoted above to `false` is enough.

Reply:
@vit9696 or in plist or ssdt and a function that can be useful if you think it's not useful i close the pr i don't want to create disagreement ☹️

I created this pr to give users with different layout keyboards used to using windows and macos to use this ssdt that modifies the keys, just that.